src/Test/Factories.php

```diff
@@ -26,7 +29,16 @@ trait Factories
      * @before
      */
     #[Before]
-    public static function _bootFoundry(): void
+    public function _beforeHook(): void
+    {
+        $this->_bootFoundry();
+        $this->_loadDataProvidedProxies();
+    }
```

Comment:
I had an issue with the order of execution, so I grouped the two method under the same hook